RLTest/__main__.py

```diff
@@ -225,8 +228,16 @@ def __init__(self):
         sys.path.append(self.args.tests_dir)
 
         self.tests = []
-
         self.currEnv = None
+        self.loader = TestLoader(filter=self.args.test_name)
+        if self.args.test:
+            self.loader.load_spec(self.args.test)
```

Comment:
In case of specifying --tests-dir then this should look inside the give directory ..

Reply:
actually I see that it possible to specify full path as -t arg so the --tests-dir should be remove from args as well as test-name and tests-file

Reply:
Also we should change Redisearch to use -t instead of those options

Reply:
I don't want to mess around with these options yet, os that our tests don't automatically break on ci. the idea is to merge this, then change the tests in ci, then remove the legacy options.